### ISSUE TITLE
plot(df) error

### ISSUE THREAD

**Issue by No-longlong** [sfu-db/dataprep#131]
![image](https://user-images.githubusercontent.com/48305924/79629558-b2494080-8185-11ea-9dfc-480a965df2ba.png)
![image](https://user-images.githubusercontent.com/48305924/79629571-c2612000-8185-11ea-91b3-e5d8837c5858.png)

**Comment by dovahcrow** [comment]
@No-longlong Thanks for the bug report! Could you please have a try on running `pip install -U git+https://github.com/sfu-db/dataprep@fix/windows-report-permission` to install the quick fix I just pushed too see if the bug get fixed?

**Comment by sl4shxX** [comment]
@dovahcrow thanks for the bug fix! It worked for me using Windows.

**Comment by sl4shxX** [comment]
> @No-longlong Thanks for the bug report! Could you please have a try on running `pip install -U git+https://github.com/sfu-db/dataprep@fix/windows-report-permission` to install the quick fix I just pushed too see if the bug get fixed?

@dovahcrow Thanks for the bug fix! It worked for me using Windows.

**Comment by dovahcrow** [comment]
Great! I'm merging the PR and do a hotfix release

**Comment by No-longlong** [comment]
![image](https://user-images.githubusercontent.com/48305924/79767528-702a2580-8364-11ea-989d-809f6b338f7d.png)

**Comment by No-longlong** [comment]
![image](https://user-images.githubusercontent.com/48305924/79767556-79b38d80-8364-11ea-9a01-f2bed6b465a0.png)

**Comment by dovahcrow** [comment]
@No-longlong can you try this piece of code in your Jupyter to see if any errors are thrown out?
```python
from tempfile import NamedTemporaryFile
with NamedTemporaryFile(suffix=".html") as f:
    f.write(b"foo")
```

**Comment by stardust3dd** [comment]
I am still having this issue, and the quick fix does not work for me. This is the output I get on running the command:
![1](https://user-images.githubusercontent.com/43073819/81624495-ef66c280-9413-11ea-9d72-921cfef3704f.PNG)
Can you please guide me where I went wrong? I am executing cells in Jupyter notebooks as well as in VSCode.

**Comment by stardust3dd** [comment]
> 
> 
> @No-longlong Thanks for the bug report! Could you please have a try on running `pip install -U git+https://github.com/sfu-db/dataprep@fix/windows-report-permission` to install the quick fix I just pushed too see if the bug get fixed?

I am still having this issue, and the quick fix does not work for me. This is the output I get on running the command:
![1](https://user-images.githubusercontent.com/43073819/81624495-ef66c280-9413-11ea-9d72-921cfef3704f.PNG)
Can you please guide me where I went wrong? I am executing cells in Jupyter notebooks as well as in VSCode.

**Comment by dovahcrow** [comment]
@No-longlong have you tried this?

> @No-longlong can you try this piece of code in your Jupyter to see if any errors are thrown out?
> ```python
> from tempfile import NamedTemporaryFile
> with NamedTemporaryFile(suffix=".html") as f:
>     f.write(b"foo")
> ```

The branch you were trying is already got merged (as confirmed by @sl4shxX). I guess the bug you have is due to the permission issue. Please try the code piece above to confirm my guess.

**Comment by stardust3dd** [comment]
> 
> 
> @stardust3dd have you tried this?
> 
> > @No-longlong can you try this piece of code in your Jupyter to see if any errors are thrown out?
> > ```python
> > from tempfile import NamedTemporaryFile
> > with NamedTemporaryFile(suffix=".html") as f:
> >     f.write(b"foo")
> > ```
> 
> The branch you were trying is already got merged (as confirmed by @sl4shxX). I guess the bug you have is due to the permission issue. Please try the code piece above to confirm my guess.

Yes, I do have the permission issue. Apologies for not clarifying that earlier. Above code produces no error for me.
![1](https://user-images.githubusercontent.com/43073819/81625162-ac0d5380-9415-11ea-91dd-73dc56290aa5.PNG)

**Comment by dovahcrow** [comment]
@stardust3dd Wierd. If the code piece runs, then you do not have the permission issue (Windows's permission error can mean lots of things, one of them is double file opening which is fixed in `fix/windows-report-permission`).

May I have your full error message (including the stack trace)?

**Comment by stardust3dd** [comment]
Thanks for bearing with me, @dovahcrow . Here is the stack trace:
![1](https://user-images.githubusercontent.com/43073819/81625768-779a9700-9417-11ea-8842-7ad20f754698.png)

dataprep version is 0.2.3:
![1](https://user-images.githubusercontent.com/43073819/81625854-b3356100-9417-11ea-9859-ff996a401aea.png)

**Comment by dovahcrow** [comment]
@stardust3dd I see. Can you upgrade dataprep to `0.2.5`? We landed the fix for windows in `0.2.4` (https://github.com/sfu-db/dataprep/releases).

**Comment by stardust3dd** [comment]
> 
> 
> @stardust3dd I see. Can you upgrade dataprep to `0.2.5`? We landed the fix for windows in `0.2.4` (https://github.com/sfu-db/dataprep/releases).

Thank you sir. Works just fine now.

**Comment by dovahcrow** [comment]
Close due to inactivity.
